src/pages/profiles/ProfileDetailHeader.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, useState } from "react";
+import { Link } from "react-router-dom";
+import DeleteProfileBtn from "./actions/DeleteProfileBtn";
+import { LxdProfile } from "types/profile";
+import ProfileRename from "pages/profiles/forms/ProfileRename";
+import { Tooltip } from "@canonical/react-components";
+
+interface Props {
+  name: string;
+  profile?: LxdProfile;
+  project: string;
+}
+
+const ProfileDetailHeader: FC<Props> = ({ name, profile, project }) => {
+  const [isRename, setRename] = useState(false);
+  const canRename = profile && profile.name !== "default";
+
+  return (
+    <div className="p-panel__header">
```

Comment:
Can we pull out a generic component here?
To contain the style overrides for the `p-breadcrumbs`, currently we override them without any condition. I think it would be good if we can put the overrides under a selector, where the selector is the root of a reused generic component.
Though it might be challenging to find a good interface for the generic component that is not too overloaded.

Reply:
This PR is growing too big already. Feel free to explore this possibility out of the scope of this PR.